packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -234,9 +235,9 @@ function useMediaUploadQueue() {
 
   const trimVideoItem = useCallback(
     async (item) => {
-      const { id, file, additionalData = {}, trimData } = item;
+      const { id, elementId, file, additionalData = {}, trimData } = item;
 
-      startTrimming({ id });
+      startTrimming({ elementId });
```

Comment:
I don‘t think we wanna change that; this should still operate on the queue item ID.

Reply:
Okay -- just noticed your comments so I'll make the other updates + revert this

Reply:
reverted here - https://github.com/GoogleForCreators/web-stories-wp/pull/10712/commits/42cb79b0040d3c77165c0879780527b852f5ddc5